geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisKey.java

```diff
@@ -29,41 +28,23 @@
 import org.apache.geode.internal.serialization.DeserializationContext;
 import org.apache.geode.internal.serialization.KnownVersion;
 import org.apache.geode.internal.serialization.SerializationContext;
-import org.apache.geode.redis.internal.executor.cluster.CRC16;
 import org.apache.geode.redis.internal.executor.cluster.RedisPartitionResolver;
 
 public class RedisKey implements DataSerializableFixedID {
 
-  private int crc16;
+  private int slot;
```

Comment:
did you consider changing this field to "short" since its value will always be in the range 0..REDIS_SLOTS-1?

Reply:
done